### ISSUE TITLE
Used by and require links broken with groups

### ISSUE THREAD

**Issue by jwa107** [SassDoc/sassdoc#448]
sassdoc v 2.1.20

If I define a group:
e.g.
```
////
/// @group shared
/// @access public
////
```

Then in the outputted docs the URL of the items get prefixed with "shared"  (e.g. `/docs/scss/#shared-mixin-font`). However if a mixin has a "Used By" or a "Requires" section, the link does not contain the group prefix (e.g. `/docs/scss/#mixin-font`). This also applies to clicking on items returned by the search. Note that the prefixed URLs (with "shared-") are the ones that actually work

Additional info....
Here is the gulp task I'm running:

```
gulp.task('sassdoc', function () {
    var options = {
        dest: 'docs/scss',
        display: {
            access: ['public', 'private'],
            alias: true,
        },
        groups: {
            'shared': 'Global',
        },
        basePath: '[redacted]',
    };

    return gulp.src('Shared/Styles/Partials/*.scss')
        .pipe(sassdoc(options));
});
```

**Comment by pascalduez** [comment]
Hi,

sassdoc version, sassdoc-theme-default version please ?

**Comment by jwa107** [comment]
@pascalduez I added the version to my post, I'm not sure about the theme-default version, but I have not made any changes to the theme, it's whatever cam with sassdoc.

**Comment by pascalduez** [comment]
Should be fixed as of `sassdoc-theme-default@2.4.2`.
